serge/configs/caps_config.serge

```diff
@@ -20,6 +20,8 @@ jobs
     {
         @inherit                   base_config.serge#common-settings/job
 
+        db_source                   DBI:SQLite:dbname=../db/capitalizedTranslations.db3
```

Comment:
@sladymon one thing to note, I have split the `db_source` for each of the config files into their own database. The next time serge will create the new database in `serge/db` directory and begin using the appropriate translation memory for each of the config files.